lms/djangoapps/courseware/tests/test_course_survey.py

```diff
@@ -77,7 +77,7 @@ def _assert_survey_redirect(self, course):
         """
         Helper method to assert that all known redirect points do redirect as expected
         """
-        for view_name in ['courseware', 'info', 'progress']:
+        for view_name in ['courseware', 'openedx.course_experience.course_home', 'progress']:
```

Comment:
Can you just confirm and ensure that the course home version of these tests weren't added elsewhere, and that these are not turning into duplicate tests?

NOTE: Ideally we would have had two tests and we should have just been overriding the flag to keep these tests working against Course Info.  However, if we only had a single test and we don't want to expand our suite, it seems reasonable to convert these to work against the newer page so when the flag and Course Info page get deleted, these tests continue to pass.

Reply:
I don't think there's a way to know for sure but as far as I can tell these tests were not duplicated elsewhere